packages/cli/src/commands/multi-chain/deploy.ts

```diff
@@ -0,0 +1,295 @@
+              indexerAvailableVersions,
+              true
+            );
+          } else {
+            indexerVersions[validator.chainId] = indexerAvailableVersions[0];
+          }
+        } catch (e) {
+          throw new Error(chalk.bgRedBright('Indexer version is required'));
+        }
+      }
+
+      indexerVersions[validator.chainId] = addV(indexerVersions[validator.chainId]);
+
+
+      if (!endpoints[validator.chainId]) {
+        if (flags.useDefaults) {
+          throw new Error(chalk.red('Please ensure a valid is passed using --endpoint flag with syntax chainId:rpc_endpoint,chainId2:rpc_endpoint2...'));
+        }
+
+        endpoints[validator.chainId] = await promptWithDefaultValues(cli, `Enter endpoint for ${multichainProjectPath}`, undefined, null, true);
+      }
+
+
+      if (!dictionaries[validator.chainId])
+      {
+        const validateDictEndpoint = processEndpoints(await dictionaryEndpoints(ROOT_API_URL_PROD), validator.chainId);
+        if (!flags.useDefaults && !validateDictEndpoint) {
+          dictionaries[validator.chainId] = await promptWithDefaultValues(cli, `Enter dictionary for ${multichainProjectPath}`, validateDictEndpoint, null, false);
+        } else {
+          dictionaries[validator.chainId] = validateDictEndpoint;
+        }
+      }
+
+
+      chains.push(
+        {
+          cid: multichainProjectCid,
+          dictEndpoint: dictionaries[validator.chainId],
+          endpoint: endpoints[validator.chainId],
+          indexerImageVersion: indexerVersions[validator.chainId],
+          indexerAdvancedSettings: {
+            indexer: indexerAD,
+          },
+          extraParams: flags.indexerWorkers ?
+            {
+              workers: {
+                num: flags.indexerWorkers,
+              },
+            } :
+            {}
+        },
+      );
+    }
+
+
+    if (projectInfo !== undefined) {
+      await updateDeployment(
+        org,
+        projectName,
+        projectInfo.id,
+        authToken,
+        ipfsCID,
+        queryVersion,
+        queryAD,
+        chains,
+        ROOT_API_URL_PROD
+      );
+      this.log(`Project: ${projectName} has been re-deployed`);
+    } else {
+      this.log('Deploying SubQuery project to Hosted Service');
+      const deploymentOutput = await createDeployment(
+        org,
+        projectName,
+        authToken,
+        ipfsCID,
+        queryVersion,
+        flags.type,
+        queryAD,
+        chains,
+        ROOT_API_URL_PROD
+      ).catch((e) => this.error(e));
+
+      // Managed service does some interesting things: `disableStoreCacheAsync: true` -> `--store-cache-async=true`, we are just inverting it to resolve logging confusion.
+      if (
+        deploymentOutput.configuration.config.indexer &&
+        deploymentOutput.configuration.config.indexer.disableStoreCacheAsync === false
+      ) {
+        deploymentOutput.configuration.config.indexer.disableStoreCacheAsync = true;
+      }
```

Comment:
This has now been removed. See this for other changes https://github.com/subquery/subql/pull/2282/files